drivers/ethernet/eth_mcux.c

```diff
@@ -284,9 +285,9 @@ static void eth_mcux_phy_event(struct eth_context *context)
 	switch (context->phy_state) {
 	case eth_mcux_phy_state_initial:
 #ifdef CONFIG_SOC_SERIES_IMX_RT
-		ENET_StartSMIRead(ENET, phy_addr, PHY_CONTROL2_REG,
+		ENET_StartSMIRead(context->base, phy_addr, PHY_CONTROL2_REG,
```

Comment:
Since context->base is not volatile the compiler will optimize the dereferencing of the value when necessary. Or do you suggest to use a local variable for readability?

Reply:
mostly readability, i assume the compiler is smart enough to optimize.

Reply:
I'm not convinced about the benefit for readability, but if you insist I can do it and if I do it for `context->base` I think it make sense to do it for `context->phy_addr` as well.

Reply:
@agansari Please tell me If you want me to do the changes or not.

Reply:
@armandciejak i'm okay with your current implementation this is why I gave it a +1